src/components/Products/AddItemButton.spec.js

```diff
@@ -0,0 +1,20 @@
+import { getByRole, screen } from '@testing-library/dom'
+import { render } from '@testing-library/react'
+import AddItemButton from './AddItemButton'
+import { MemoryRouter as Router } from 'react-router-dom'
+
+describe('AddItemButton', () => {
+  it('renders a button with a link', () => {
+    render(
+      <Router>
+        <AddItemButton />
+      </Router>
+    )
+
+    const button = screen.getByRole('button')
+    expect(button).toBeInTheDocument()
+
+    const link = screen.getByText('Add Item')
+    expect(link).toBeInTheDocument()
+  })
+})
```

Comment:
The test looks nice, maybe consider to test if the links are working as well.

Reply:
Hm, you can't really test if the links are working! Since we are using `react-router-dom` we can be certain that they have tested their components thoroughly. For us it is enough to see _that_ the links are rendered.